metadatastore/odm_templates.py

```diff
@@ -23,7 +23,7 @@ class BeamlineConfig(DynamicDocument):
     config_params = DictField(required=False, unique=False)
     uid = StringField(required=True, unique=True)
     time = FloatField(required=True)
-    meta = {'indexes': ['-_id'], 'db_alias': ALIAS}
+    meta = {'indexes': ['-_id', '-uid'], 'db_alias': ALIAS}
```

Comment:
Of course it matters. The daemon has a priority buffer that keeps the most recent stuff that is denoted by "-". Eric does the right thing by ensuring it is indexed properly. Also, meta has tons of neat features we might use in the future.